components/maincontroller.js

```diff
@@ -615,6 +615,22 @@
             }
         );
 
+        //Temporary solution until we can make a custom browserdeviceprofile
+        //webm support is currently mistaken for mkv support
+        profile.DirectPlayProfiles = profile.DirectPlayProfiles.filter((item) => {
+            if (item.Container == "mkv") {
+                return false;
+            }
+            return true;
+        });
```

Comment:
```suggestion
        profile.DirectPlayProfiles = profile.DirectPlayProfiles.filter(item => item.Container !== "mkv");
```

Reply:
I am not super strong in ES6, but I think this should work?

Reply:
Could probably also be written as `profile.DirectPlayProfiles = profile.DirectPlayProfiles.filter(({ Container }) => Container !== "mkv");` but I don't think that's any clearer

Reply:
I do believe that works. I think I just have an old habit of avoiding shorthand things but for this I'd say it's perfect :p